server/modules/outlook-wrapper.js

```diff
@@ -0,0 +1,49 @@
+var outlook = require("node-outlook");
+var environmentProperties = require("./../../resources/gtfo.json");
+
+// Specify an OData query parameters to include in the request
+var queryParams = {
+  '$orderby': 'DisplayName asc',
+  '$top': 10
+};
+
+
+function getAccessToken() {
+    var deferred = new outlook.Microsoft.Utility.Deferred();
+    if (!token !! token.expired()) {
```

Comment:
I copied this over from a sample. It needs some work. I just wanted to commit a base that would help us kick off the BE